service/history/historyEngine.go

```diff
@@ -2674,6 +2684,18 @@ func (e *historyEngineImpl) ResetTimerQueue(
 	return err
 }
 
+func (e *historyEngineImpl) ResetCrossClusterQueue(
+	ctx context.Context,
+	clusterName string,
+) error {
+	if e.crossClusterProcessor == nil {
```

Comment:
Do you want to add a comment to remind removing this check when the processor is implemented?

Reply:
Makes sense. I added a TODO at the place where`errQueueNotInitialized` is defined.